InfiniteLoopLMSSQL/src/com/st/lmssql/daoImp/BookCopiesDaoImp.java

```diff
@@ -0,0 +1,90 @@
+package com.st.lmssql.daoImp;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
+
+import com.st.lmssql.dao.BookCopiesDao;
+import com.st.lmssql.models.BookCopies;
+
+public class BookCopiesDaoImp implements BookCopiesDao {
+
+	private Connection con;
+	
+	public BookCopiesDaoImp(Connection con) {
+		this.con = con;
+	}
+	
+	@Override
+	public void add(BookCopies obj) throws SQLException {
+		String query = "INSERT INTO tbl_book_copies (bookId, branchId, noOfCopies) VALUES (?,?,?)";
+		
+		PreparedStatement pstmt = con.prepareStatement(query);
+		pstmt.setInt(1, obj.getBookId());
+		pstmt.setInt(2, obj.getBranchId());
+		pstmt.setInt(3, obj.getNoOfCopies());
+		pstmt.executeUpdate();
+	}
+	
+	@Override
+	public int getNoOfCopies(int bookId, int branchId) throws SQLException {
+		int noOfCopies = 0;
+		String query = "SELECT * " +

```

Comment:
This would only be useful if you are trying not to go over the recommended 50 to 80 characters until you want a new line, or you want to concatenate a variable